tests/rules/method.js

```diff
@@ -219,6 +220,49 @@ eslintTester.run("method", rule, {
             code: "(e = node.insertAdjacentHTML('beforebegin', '<s>safe</s>'))()",
             parserOptions: { ecmaVersion: 6 },
         },
+
+        // Typescript support tests
+        // raw strings can be assigned to innerHTML
+        {
+            code: "(options as HTMLElement).innerHTML = 'evil';",
```

Comment:
please don't call it evil here. Call it `safe` or `<s>safe</s>` like above

Reply:
Done - oops!